docs/reference/ml/anomaly-detection/ml-configuring-transform.asciidoc

```diff
@@ -1,14 +1,14 @@
 [role="xpack"]
 [[ml-configuring-transform]]
-= Transforming data with script fields
+= Altering data with runtime mappings
 
-If you use {dfeeds}, you can add scripts to transform your data before
-it is analyzed. {dfeeds-cap} contain an optional `script_fields` property, where
-you can specify scripts that evaluate custom expressions and return script
-fields.
+If you use {dfeeds}, you can use runtime mappings to alter your data before it 
+is analyzed. You can add an optional `runtime_mappings` property to your 
+{dfeeds}, where you can specify field types and scripts that evaluate custom 
+expressions.
 
-If your {dfeed} defines script fields, you can use those fields in your
-{anomaly-job}. For example, you can use the script fields in the analysis
+If your {dfeed} defines runtime fields, you can use those fields in your
+{anomaly-job}. For example, you can use the runtime fields in the analysis
 functions in one or more detectors.
```

Comment:
The runtime fields documentation describes some compromises (e.g. they "can impact search performance based on the computation defined in the runtime script"). I presume the same performance impact exists when they're used in datafeeds. Should we also call that out here too?